Cannot find module

On latest version, when using Node 12:

```
❯ yarn -s prettier -h
internal/modules/cjs/loader.js:796
    throw err;
    ^

Error: Cannot find module 'comment-parser/lib'
Require stack:
- /Users/jasonkuhrt/projects/prisma/nexus-prisma/node_modules/@homer0/prettier-plugin-jsdoc/src/fns/getParsers.js
- /Users/jasonkuhrt/projects/prisma/nexus-prisma/node_modules/@homer0/prettier-plugin-jsdoc/src/fns/getPlugin.js
- /Users/jasonkuhrt/projects/prisma/nexus-prisma/node_modules/@homer0/prettier-plugin-jsdoc/src/index.js
- /Users/jasonkuhrt/projects/prisma/nexus-prisma/node_modules/prettier/index.js
- /Users/jasonkuhrt/projects/prisma/nexus-prisma/node_modules/prettier/bin-prettier.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:793:17)
    at Function.Module._load (internal/modules/cjs/loader.js:686:27)
    at Module.require (internal/modules/cjs/loader.js:848:19)
    at require (internal/modules/cjs/helpers.js:74:18)
    at Object.<anonymous> (/Users/jasonkuhrt/projects/prisma/nexus-prisma/node_modules/@homer0/prettier-plugin-jsdoc/src/fns/getParsers.js:2:34)
    at Module._compile (internal/modules/cjs/loader.js:955:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:991:10)
    at Module.load (internal/modules/cjs/loader.js:811:32)
    at Function.Module._load (internal/modules/cjs/loader.js:723:14)
    at Module.require (internal/modules/cjs/loader.js:848:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/Users/jasonkuhrt/projects/prisma/nexus-prisma/node_modules/@homer0/prettier-plugin-jsdoc/src/fns/getParsers.js',
    '/Users/jasonkuhrt/projects/prisma/nexus-prisma/node_modules/@homer0/prettier-plugin-jsdoc/src/fns/getPlugin.js',
    '/Users/jasonkuhrt/projects/prisma/nexus-prisma/node_modules/@homer0/prettier-plugin-jsdoc/src/index.js',
    '/Users/jasonkuhrt/projects/prisma/nexus-prisma/node_modules/prettier/index.js',
    '/Users/jasonkuhrt/projects/prisma/nexus-prisma/node_modules/prettier/bin-prettier.js'
  ]
}
```

When using Node 16:

```
❯ yarn -s prettier -h
node:internal/modules/cjs/loader:482
      throw e;
      ^

Error [ERR_PACKAGE_PATH_NOT_EXPORTED]: Package subpath './lib' is not defined by "exports" in /Users/jasonkuhrt/projects/prisma/nexus-prisma/node_modules/comment-parser/package.json
    at new NodeError (node:internal/errors:363:5)
    at throwExportsNotFound (node:internal/modules/esm/resolve:335:9)
    at packageExportsResolve (node:internal/modules/esm/resolve:560:3)
    at resolveExports (node:internal/modules/cjs/loader:476:36)
    at Function.Module._findPath (node:internal/modules/cjs/loader:516:31)
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:913:27)
    at Function.Module._load (node:internal/modules/cjs/loader:772:27)
    at Module.require (node:internal/modules/cjs/loader:999:19)
    at require (node:internal/modules/cjs/helpers:93:18)
    at Object.<anonymous> (/Users/jasonkuhrt/projects/prisma/nexus-prisma/node_modules/@homer0/prettier-plugin-jsdoc/src/fns/getParsers.js:2:34) {
  code: 'ERR_PACKAGE_PATH_NOT_EXPORTED'
}
```

--- a/packages/public/prettier-plugin-jsdoc/package.json
+++ b/packages/public/prettier-plugin-jsdoc/package.json
@@ -20,7 +20,7 @@
     "access": "public"
   },
   "dependencies": {
-    "comment-parser": "^1.1.5",
+    "comment-parser": "^1.2.3",
     "prettier": "^2.3.2",
     "ramda": "0.27.1"
   },

--- a/packages/public/prettier-plugin-jsdoc/src/fns/getParsers.js
+++ b/packages/public/prettier-plugin-jsdoc/src/fns/getParsers.js
@@ -1,5 +1,5 @@
 const R = require('ramda');
-const { parse: commentParser } = require('comment-parser/lib');
+const { parse: commentParser } = require('comment-parser/lib/index.cjs');
 const babelParser = require('prettier/parser-babel');
 const flowParser = require('prettier/parser-flow');
 const tsParser = require('prettier/parser-typescript');

